.storybook/config.js

```diff
@@ -1,16 +1,70 @@
 import React, { Component } from 'react'
+import { Table, Tag } from 'antd';
+import { Provider } from 'mobx-react';
+import { pickBy } from 'lodash';
+
 import { configure, addDecorator } from '@storybook/react';
+import { withInfo } from "@storybook/addon-info";
+
 import 'antd/dist/antd.css';
+import './stories.css';
 
-const styles = {
-  margin: '15px',
-};
-const CenterDecorator = (storyFn) => (
-  <div style={styles}>
-    { storyFn() }
+const PropsTable = (props) => (
+  <Table
+    dataSource={props.propDefinitions}
+    pagination={false}
+    bordered
+    rowKey='property'
+    columns={[
+      {
+        dataIndex: 'property',
+        sorter: (a, b) => (a.property).localeCompare(b.property),
+        title: 'Prop'
+      },
+      {
+        dataIndex: 'propType.name',
+        sorter: (a, b) => (a.propType.name).localeCompare(b.propType.name),
+        title: 'Type'
+      },
+      {
+        dataIndex: 'required',
+        defaultSortOrder: 'descend',
+        render: value => value ? <Tag color="red">Yes</Tag> : <Tag>No</Tag>,
+        sorter: (a, b) => (a.required === b.required)? 0 : a.required? 1 : -1,
+        title: 'required',
+      },
+    ].map(row => ({ ...row, key: row.dataIndex }))}
+  />
+);
+
+const PageDecorator = (storyFn) => (
+  <div style={{ margin: '15px' }}>
+    <Provider {...providers}>
+      { storyFn() }
+    </Provider>
   </div>
 );
-addDecorator(CenterDecorator);
+
+const providers = {
+  getOptions: (optionType) => [
+    { value: 'first', name: 'first' },
+    { value: 'second', name: 'second' },
+    { value: 'third', name: 'third' },
+  ],
+};
+
+const ComponentDecorator = (storyFn) => (
+    <div className='story-component'>
+      { storyFn() }
+    </div>
+);
+
+addDecorator(ComponentDecorator);
```

Comment:
Why do you need two decorators that both call `storyFn()`? I'm not familiar with storybook

Reply:
The decorators stack and wrap each other. So one needs to be before the withInfo, and one after.